docs/Filtering-Details.md

```diff
@@ -0,0 +1,142 @@
+# Filtering Details
+
+## Overview
+
+This application takes an array of hashes passed to the `filters` parameter
+within a `systemd` typed source definition in your `fluent.conf` configuration
+file and then parses them into a format understood by `systemd`'s `journald`
```

Comment:
I am not sure that it is strictly true to describe this as the `journald` API...

I think "libsystemd's journal API" would be slightly more accurate description...